src/group_theory/subgroup.lean

```diff
@@ -779,6 +779,11 @@ lemma mem_map {f : G →* N} {K : subgroup G} {y : N} :
   y ∈ K.map f ↔ ∃ x ∈ K, f x = y :=
 mem_image_iff_bex
 
+@[to_additive]
+lemma mem_map_of_mem {K : subgroup G} (f : G →* N) {x : G} (hx : x ∈ K) :
+  f x ∈ subgroup.map f K :=
+mem_image_of_mem f hx
```

Comment:
To match `submonoid`:
```suggestion
@[to_additive]
lemma mem_map_of_mem {K : subgroup G} (f : G →* N) (x : K) :
  f x ∈ subgroup.map f K :=
mem_image_of_mem f x.prop
```

If that's not helpful for LTE, then maybe add both versions? (and for submonoid too)

Reply:
Done!